### PR TITLE
Spawning at the Odin

### PR DESCRIPTION
**Pull request by LordFowl**
+ Latejoiners will spawn at the Odin, in an elevator shaft in a segregated section of the station away from round-end.
+ Latejoiners will spawn in civilian clothes equated to their paygrade. (List currently is extremely rudimentary.)
+ Latejoiners will get their implants activated, their departmental memories inserted, and their equipment delivered by the auto-locker machines. It is at this point that they will be announced to the crew.
+ Latejoiners will then proceed to the arrivals shuttle. As soon as any living being steps on the shuttle, it will begin counting down to launch in 30 seconds. If there is no living being on it when it is about to launch, it will cancel.
+ The shuttle will be in transit for one minute. When it arrives it will wait a minute, and then attempt to launch back to the Odin. If there are any living beings on it, it will cancel, announce this to the crew, and then try again in another minute.
+ Anyone who remains in the spawn area for longer than 15 minutes will be despawned if they're SSD. If they are not SSD they will be teleported to Aurora's cryo. If Aurora's cryo is somehow destroyed they'll be despawned.
+ Spawning area itself also has its own cryo, if for whatever reason that is needed.

In playtesting this I noticed a bug where shuttles just don't work. However, as that was not caused by this PR AFAIK I did not think it prudent to let it stop me from making it. Isolating this bug should become a sort of priority objective, along with all the other shitty fucking bugs.

### CODE REVIEW

**Comment by LordFowl** [comment]
@skull132 Leaving this up for review because of it's significant code content.

**Comment by Fire-and-Glory** [comment]
Is there something to stop beastmen from spawning bare-foot if they didn't bother getting the lizard-boots?

**Comment by LordFowl** [comment]
@Fire-and-Glory Due to the nature of how sandal equipping is handled for the lesser races, it should be largely unaffected by my PR.

File: code/controllers/arrival_shuttle_controller.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
+var/global/datum/arrival_shuttle_controller/arrival_shuttle
+
+/datum/arrival_shuttle_controller
```
Comment (Lohikar) [inline code/controllers/arrival_shuttle_controller.dm]: Why isn't this `/datum/controller/arrival_shuttle`?

Comment (LordFowl) [inline code/controllers/arrival_shuttle_controller.dm]: No, the actual controller is in arrivalShuttle.dm, a la the emergency shuttle prototype.

**Pull request merged**